Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,338 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace test1
+{
+    public class Program
+    {
+        const int MINIMO_STOCK = 1000;
+
+        static void Main(string[] args)
+        {
+            List<Customer> customersList = new List<Customer>();
+            List<Product> productsList = new List<Product>();
+            List<Seller> sellerList = new List<Seller>();
+            List<Sale> salesList = new List<Sale>();
+
+            int optionMenu = 0;
+            bool answerMenu;
+            int numberOpcion;
+            while (optionMenu != 6)
+            {
+                do
+                {
+                    Console.WriteLine("--------------------");
+                    Console.WriteLine("Billing Menu");
+                    Console.WriteLine("--------------------");
+                    Console.WriteLine("1. Sales Registration");
+                    Console.WriteLine("2. Customers Registration");
+                    Console.WriteLine("3. Seller Registration");
+                    Console.WriteLine("4. Products Registration");
+                    Console.WriteLine("5. Reports");
+                    Console.WriteLine("6. Exit");
+                    Console.WriteLine("Select an option");
+                    answerMenu= Int32.TryParse(Console.ReadLine(), out numberOpcion);
+                }   
+                    while (answerMenu == false);

```

Comment:
Here is a wrong extra tab.

Reply:
also, when in the condition you have a bool variable, the best way would be
while (!answerMenu)